### PR TITLE
Ping graph for firewall

### PR DESCRIPTION
**Pull request by stephdl**
Make a ping graph inside the dashboard of firewall

Screenshot - 2021-09-29T165138 202
Screenshot - 2021-09-29T165116 627

### CODE REVIEW

File: api/dashboard/charts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    while (<FH>) {
+    chomp;
+    my @lines = split("\r\n");
+        foreach my $line (@lines) {
+            my @values = split(":",$line);
+            next unless $values[0] =~ m/\d/;
+            next unless $values[1] =~ m/\d/;
+            $values[1] =~ s/ //g;
+            # be sure to output a numeric value
+            $values[1] += 0;
```
Comment (gsanchietti) [inline api/dashboard/charts:53]: Also the timestamp be an integer:
```suggestion
            $values[1] += 0;
            $values[0] = int($values[0]);
```

**Review by gsanchietti — COMMENTED** [review]
Thank you!

This is a great proof of concept for the work on the new Firewall dashboard!
I didn't know about `rrdtool fetch`: it's fast ans easy to parse :+1:

File: ui/src/views/Dashboard.vue [hunk 2]
```diff
@@ -706,7 +752,114 @@ export default {
...
+    toggleCharts() {
+      this.view.chartsShowed = !this.view.chartsShowed;
+      if (this.view.chartsShowed) {
+        this.initPingChart();
+      }
+    },
+    initPingChart() {
+      var context = this;
+      nethserver.exec(
+        ["nethserver-firewall-base//dashboard/charts"],
```
Comment (edospadoni) [inline ui/src/views/Dashboard.vue:765]:
```suggestion
        ["nethserver-firewall-base/dashboard/charts"],
```

**Review by edospadoni — CHANGES_REQUESTED** [review]
Except a little typo, the PR looks good to me :+1:

**Comment by gsanchietti** [comment]
Replaced by #204

**Pull request closed without merging**